packages/coordinator/src/services/translation/response/common.ts

```diff
@@ -187,37 +187,27 @@ export function orderBundleResources(r1: fhir.Resource, r2: fhir.Resource): numb
 }
 
 function getSortIndex(resource: fhir.Resource) {
-  let sortIndex = 0
   switch (resource.resourceType) {
     case "MessageHeader":
-      sortIndex = 0
```

Comment:
Looks like the complaint was for assigning 0 to sortIndex after already assigning it on line 190